src/service_alerts.rs

```diff
@@ -41,9 +67,17 @@ pub enum Alerts {
     LatencyMillisecondsP99,
 }
 
+#[derive(Debug, Serialize, Deserialize, Clone, JsonSchema, PartialEq, Eq, Hash)]
+#[serde(rename_all = "camelCase")]
+pub enum MiscAlerts {
+    AllReplicasDown,
```

Comment:
There are some alerts that need no configuration (other than knowing which deployment to use metrics for), we could potentially have a `defaultMetrics` setting  down the line to help people with sane defaults

Reply:
Default metrics are worth thinking about, in that case I would probably be inclined to make the CRD specification simply include an option to disable default metrics for teams that want that:
```yaml
apiVersion: cactuar.rs/v1
kind: ServiceAlert
metadata:
  name: example-service-alert
spec:
  commonLabels:
    origin: cloud
    owner: bar
  deploymentName: best-service-eu
  alerts:
     disableDefaults: true
     ...snip...
```
`disableDefaults` could be either on the `alerts` section or moved up to be part of the spec top-level.

Reply:
I like this idea a lot. let's see how far we can push this, given we're already providing opinionated alerts

Reply:
If we find we want to later we can turn disableDefaults into an object naming the specific alerts